### PR TITLE
Add listing support for policies.

### PR DESCRIPTION
**Pull request by pritesh**

### CODE REVIEW

File: romana/cmd/policy.go [hunk 1]
```diff
@@ -54,7 +56,7 @@ var policyAddCmd = &cli.Command{
 }
 
 var policyRemoveCmd = &cli.Command{
-	Use:          "remove [tenantName][policyName]",
+	Use:          "remove [tenantName][policyName1][policyName2...]",
 	Short:        "Remove a specific policy.",
 	Long:         `Remove a specific policy.`,
```
Comment (jbrendel) [inline romana/cmd/policy.go]: Why does this line use a different quote?

    `     vs.   "

?

File: romana/cmd/policy.go [hunk 2]
```diff
@@ -136,7 +138,7 @@ func policyAdd(cmd *cli.Command, args []string) error {
 		}
 		fmt.Printf(util.JSONIndent(string(body)))
 	} else {
-		if resp.StatusCode == 201 {
+		if resp.StatusCode == http.StatusCreated {
```
Comment (jbrendel) [inline romana/cmd/policy.go]: Wait! Do we post a Kubernetes policy definition to the Kubernetes API?

Our policy command is supposed to send a policy definition to the policy manager, which then turns around and sends this to the policy agents.

Not to Kubernetes.

We need this command in the case of OpenStack, where we don't properly integrate and we need our tools to set policies.

In the case of Kubernetes the user defines policies via the Kubernetes API and we pick this up along the way.

**Comment by jbrendel** [comment]
After conversation with Pritesh: This tool talks to the k8s API, instead of the policy manager. That's just a place holder once we have our policy manager in place. As soon as that happens, this tool here will be changed.

File: romana/cmd/policy.go (hunk 1 above)
Comment (pritesh) [inline romana/cmd/policy.go]: ` preserves lin breaks and any content in long does that.

**Pull request merged**